Erro ao ler o trace: open /.traces/access_trace_aleatorio.csv: no such file or directory
